src/ui/molecules/card/index.css

```
@@ -13,3 +13,23 @@
 .card h2 {
   margin: 0;
 }
+
+.card .GitHub::before {
```

Tiny change for consistency. I prefer to keep everything lowercase in CSS.

```suggestion
.card .github::before {
```